openassessment/templates/openassessmentblock/message/oa_message_incomplete.html

```diff
@@ -1,56 +1,54 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">
-        {% trans "Assignment in progress" %}
-    </h3>
+    <h4 class="message__title">{% trans "Progress" %}</h4>
```

Comment:
@staubina Now that the actual status has been moved into the message body, this message title could actually be the same as the other statuses, i.e. "Status", right?